src/Provider/Provider.Web/ViewModels/VacancyPreview/VacancyPreviewViewModel.cs

```diff
@@ -145,6 +146,9 @@ private bool IsEditRequired(string fieldIdentifier)
             return Review.FieldIndicators.Any(f => f.ReviewFieldIdentifier == fieldIdentifier);
         }
 
+        [MustBeTrue(ErrorMessage = "You must confirm that the information is correct before submitting.")]
```

Comment:
This isnt needed, same as above that we are always defaulting it to true for this ViewModel

Reply:
Removed in latest.